### DISCUSSION TITLE
Navn og versjon på EPJ system

### DISCUSSION THREAD

**Discussion by AMJohnsen** [NorskHelsenett/Tillitsrammeverk#56] [Ideas]
Jeg ser at navn og versjon på klinisk fagsystem ikke er en del av informasjonselementene, men anser det som viktig @steinarnoem. Er dette noe som bør inn, og hvorfor er det ikke med allerede? Har det blitt diskutert?

**Comment by steinarnoem** [comment]
Hei! Jeg er helt enig med deg i at informasjon om programvare/system er en del av tilgangsstyrings og sporbarhetsbildet.
Informasjons- og datamodellen fokuserer på å uttrykke helsepersonellets tjenstlige behov. Jeg mener at informasjon om programvare/system naturlig hører hjemme i et access token, men tror det er lurt at dette beskrives i en annen spesifikasjon.

Jeg tror at informasjonskilden for informasjon om programvare/system vil være HelseID (igjennom selvbetjening eller medlemstjenesten), basert på en erklæring fra konsumenten.

**Comment by micer2004** [comment]
Er ikke versjon på et klinisk fagsystem litt gammeldags @AMJohnsen, referer til diskusjonene vi hadde tidligere i de siste ukene. 

Kan bli et potensielt hinder for moderne systemer med mulighet for CI/CD med jevnlige deploy. Hva er verdi med å vite versjonsnr på klinisk system vs. tillitt til bruksvilkår som aktøren har signert?

**Reply by AMJohnsen** [reply]
jeg er ingen sikkerhetsekspert @micer2004 - men her tenker jeg at det kan være fint av flere grunner;
- feilsøking i tilfelle noe skjer
- ved sikkerhetsbrudd så vet vi hvilke versjon av systemet som faktisk utgjør risikoen (i tilfelle noe nytt har blitt introdusert)

systemer med hyppige deploys bør vel ikke ha noen problemer med å sende versjon? Jeg forstår ikke problemstillingen, men jeg er villig og nysgjerrig på å høre hvorfor vi ikke skal inkludere versjon?

**Reply by micer2004** [reply]
Så lenge vi ikke bruker versjoner til godkjenning av tilganger (kan hende at det er det vi gjør i dag, for noen nasjonale løsninger) så er jeg enig. Med en gang dette skal har litt breiere bruksomfang enn som INFO/DEBUG informasjon så tror jeg at vi ikke lykkes med å få dette kartlagt. Bør kanskje sees i sammenheng med "user-agent"-header hvor HelseId vil gjerne ha utfylt nyttig info om systemet
